<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Card_</class>
 <widget class="QWidget" name="Edit_Card_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditAccountId">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>200</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCardType">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCardInfo">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>421</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCreateNewCard">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create new card</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccountID">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Account Id</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDeletecard">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete card</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCustomerID">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>70</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPinNumber">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PIN number</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEditCard_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Card type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCustomerID">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Customer Id</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCardInformation">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Card information</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPinNumber">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBack">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>520</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOwnerID">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Owner Id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOwnerID">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEditCard">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit card</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
